src/callbacks/FikaLocationCallbacks.ts

```diff
@@ -16,6 +16,6 @@ export class FikaLocationCallbacks {
 
     /** Handle /fika/location/raids */
     public handleGetRaids(_url: string, info: IGetRaidConfigurationRequestData, _sessionID: string): string {
-        return this.httpResponseUtil.noBody(this.fikaLocationController.handleGetRaids(info));
+        return this.httpResponseUtil.noBody(this.fikaLocationController.handleGetRaids(info, _sessionID));
```

Comment:
rename `_sessionID` to `sessionID`

underscore is used just so the editor doesn't scream at you because it's an unused variable

do this everywhere you used an previously unused variable.

Reply:
all done